internal/pkg/docker/orchestrator/orchestrator.go

```diff
@@ -43,15 +48,22 @@ type DockerEngine interface {
 	Run(context.Context, *dockerengine.RunOptions) error
 	IsContainerRunning(context.Context, string) (bool, error)
 	Stop(context.Context, string) error
+	Build(ctx context.Context, args *dockerengine.BuildArguments, w io.Writer) error
 }
 
 const (
 	orchestratorStoppedTaskID = -1
 	pauseCtrTaskID            = 0
 )
 
-const (
-	pauseContainerURI = "public.ecr.aws/amazonlinux/amazonlinux:2023"
+const pauseContainerURI = "aws-copilot-pause"
+
+var (
+	// '+' is not a valid character in an image tag
+	pauseContainerTag = strings.Replace(version.Version, "+", "PLUS", -1)
```

Comment:
wut do we need this for ease of development? Why do we even need to build the img with a tag 🤔

Reply:
yeah it is only development🥺 i think you're right - i'll just use the `latest` tag always instead 😊